packages/foam-vscode/src/features/document-link-provider.spec.ts

```diff
@@ -120,4 +120,23 @@ describe('Document links provider', () => {
     expect(links[0].target).toEqual(OPEN_COMMAND.asURI(noteB.uri));
     expect(links[0].range).toEqual(new vscode.Range(0, 18, 0, 33));
   });
+
+  it('should support wikilink aliases in tables using escape character', async () => {
+    const fileB = await createFile('# File B');
+    const fileA = await createFile(
+      `this is a link to [[${fileB.name}\|alias]].`
```

Comment:
I think you want `[[${fileB.name}\\|alias]]` here?

Reply:
I was under the impression it was not needed to escape the escape. But will validate it.

Reply:
let me know, maybe I am confused about what you are trying to achieve here (and what the proper md and js syntax for it is)

Reply:
Yeah, first trying to get the test stable. Will let you know.

Reply:
You're correct. It should be a double escaped. Due to the flaky tests, I didn't properly notice it. The way to solve it for now is to spread the two tests apart. For some reason, they are interfering, but I don't know why.

Reply:
Scratch that... A new approach which works locally consistently is to clean the workspace after each test.